### DISCUSSION TITLE
Como crear una pasarela de pagos para Pimcore

### DISCUSSION THREAD

**Discussion by soportevexsoluciones** [pimcore/pimcore#17750] [General]
Hola soy un desarrollador de software a medida para Pimcore estoy creando complementos para diferentes plataformas, ahora quiero integrar una pasarela de pagos para Pimcore y estoy buscando una guía de como iniciar si alguien me podría ayudar, ¡muchas gracias!

**Comment by AlephLau** [comment]
@soportevexsoluciones  Nobody will answer you if you don't speak English. I'm sharing a part of the documentation for the Ecommerce Framework module with you. It includes the Checkout Manager and Payment Manager, which can help you implement a payment gateway regardless of the platform you're integrating with. Good luck! 
https://pimcore.com/docs/platform/Ecommerce_Framework/Architecture_Overview
